.github/workflows/publish-acr-image.yml

```diff
@@ -0,0 +1,80 @@
+name: Push image to ACR
+on:
+  repository_dispatch:
+    types: [ publish-mcr-image ]
+
+permissions:
+  contents: write
+  packages: write
+
+env:
+  GO_VERSION: '1.20'
+  IMAGE_NAME: 'workspace'
+
+jobs:
+  build-publish-mcr-image:
```

Comment:
A little confuse about the name. The file is publish-acr-image, and why mcr is here? or maybe your should change the file name.